dictionaries/software-terms/src/software-terms.txt

```diff
@@ -2190,3 +2190,21 @@ zip
 zlib
 ZPool
 ZPools
+builddir
```

Comment:
Please add these to coding-terms. Since software-terms has grown so much, I'm hoping to have software-terms be concepts or algorithms like "sort", "hash", "trie", and "cache". I'm trying to move the other terms to their respective files. A simple rule might be, "is it a computing term found on Wikipedia?".